_posts/2021-09-18-p2pool-released.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monero P2Pool is now live"
+summary: P2Pool centralized pool for Monero mining is live on the mainnet
+tags: [community]
+author: QuickBASIC
+---
+
+We are happy to announce that Monero P2Pool is live on the mainnet and binaries have been [released](https://github.com/SChernykh/p2pool/releases).
+
+### What is Monero P2Pool?
+
+Monero P2Pool is a peer-to-peer Monero mining pool developed from scratch by SChernykh (also known as sech1). P2Pool combines the advantages of pool and solo mining; you still fully control your Monero node and what it mines, but you get frequent payouts like on a regular pool. P2Pool as a concept was first developed for the Bitcoin blockchain but was never fully realized due to certain limitations, mainly because it had a problem with orphaned blocks which is solved in Monero P2Pool by uncle blocks.
+
+P2Pool has no central server that can be shutdown/blocked because it uses a separate blockchain to merge mine with Monero. There's no pool admin that can control what your hashrate is used for or decide who can mine on the pool and who can't --- It's permissionless!
+
+All pool blocks pay out to miners immediately. Because there is no pool wallet, funds are never in custody --- It's trustless.
+
+P2Pool uses PPLNS payout scheme, has 0% fee and 0 XMR payout fee (by design, fees can't be added). Everyone in the PPLNS window is rewarded directly via the coinbase transaction reward. Minimum payout is less than 0.0005 XMR due to the very efficient miner reward transaction format.  --- each individual miner payout takes only 38 bytes on the Monero blockchain!
+
+### Why Monero P2Pool?
+
+There's been some worry about mining pool centralization and the possibility of 51% attacks. Since atomic swaps went live, solving exchange delisting/KYC threat, mining is now the most centralized part of the Monero ecosystem. Currently, one pool accounts for almost 40% of Monero's network hashrate; this is not a stable situation long-term, because a pool admin can be pressured to do malicious things, shutdown their pool, or expropriate funds without paying miners.

```

Comment:
> solving exchange delisting/KYC threat

This is not really the case. it helps for sure, but the problem remains for xmr <-> fiat and you cannot do atomic swaps with that.

Reply:
Indirectly it still solves it. `xmr <-> btc <-> fiat` will always be possible with atomic swaps.

Reply:
That's already the case, not necessarily with atomic swaps. Anyway, it's not really important. If people prefer this phrasing i'm ok with it.

Reply:
This verbiage was actually toned down from the original text (from SChernykh). I'll leave it for now and if anyone has a direct suggestion of how to change it we can do so.